pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -396,22 +410,22 @@ type ClusterStatus struct {
 	// +optional
 	Versions ClusterVersionsStatus `json:"versions,omitempty"`
 
-	// UserName contains the name of the owner of this cluster
+	// UserName contains the name of the owner of this cluster.
 	// +optional
 	UserName string `json:"userName,omitempty"`
-	// UserEmail contains the email of the owner of this cluster
+	// UserEmail contains the email of the owner of this cluster.
```

Comment:
You could add

> This email address is bound to the cluster-admin ClusterRole inside the usercluster.

But this can be misleading, as this binding happens only once under special circumstances and changing the field in the ClusterStatus *will not* update the binding. So I'm not sure how much we want to document this field (a field I am thinking about moving to the ClusterSpec anyway, maybe, potentially not, not sure).

Reply:
The email is also made available, just like the name, as an addon templating variable, but KKP itself doesn't use it for templates and I doubt(?) that external addons use the fields either.

Reply:
I added this as second line:

```
// During cluster creation only, this field will be used to bind the `cluster-admin` `ClusterRole` to a cluster owner.
```

WDYT?